### DISCUSSION TITLE
新数据库

### DISCUSSION THREAD

**Discussion by Jeamorg** [Qexo/Qexo#273] [Ideas]
以后会支持 https://planetscale.com/ 这个数据库吗？

**Comment by am-abudu** [comment]
如果是本地部署，请参考Mysql理论可直接配置
如果是Vercel部署，请等待后续逻辑更新，我将支持Vercel下的Mysql部署

**Comment by am-abudu** [comment]
已经给予了支持。你需要配置环境变量`["MYSQL_NAME", "MYSQL_HOST", "MYSQL_PORT", "MYSQL_USER", "MYSQL_PASSWORD"]`同时将环境变量`PLANETSCALE`设置为1

**Reply by Jeamorg** [reply]
成功了

**Reply by am-abudu** [reply]
@JeamUmika 好的，将在下个版本发布后加入文档

**Reply by Fgaoxing** [reply]
@am-abudu 再加个pg数据库把
